scripts/exrrfs_bufrsnd.sh

```diff
@@ -144,9 +144,8 @@ cyc=$hh
 #
 #-----------------------------------------------------------------------
 #
-dom=conus
+dom=na
```

Comment:
Can `dom` take values from a variable set in the config.sh file so that both CONUS and NA runs are able to use this script?

Reply:
@guoqing-noaa Went in a different direction and just eliminated the domain information from the script.

Reply:
It looks great! Thanks for the changes!